<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PendingDialog</class>
 <widget class="QDialog" name="PendingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="loans_tab">
      <attribute name="title">
       <string>Loans</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="loans_table">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <column>
          <property name="text">
           <string>UOMe ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Borrower</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Amount</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="debts_tab">
      <attribute name="title">
       <string>Debts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="debts_table">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <column>
          <property name="text">
           <string>UOMe ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Loaner</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Amount</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="select_all_button">
       <property name="text">
        <string>Select All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="select_none_button">
       <property name="text">
        <string>Select None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="button_box">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>PendingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>PendingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_none_button</sender>
   <signal>clicked()</signal>
   <receiver>PendingDialog</receiver>
   <slot>select_none()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_all_button</sender>
   <signal>clicked()</signal>
   <receiver>PendingDialog</receiver>
   <slot>select_all()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>select_none()</slot>
  <slot>select_all()</slot>
 </slots>
</ui>
